pkg/controller/manager/manager_controller_test.go

```diff
@@ -223,19 +215,33 @@ var _ = Describe("Manager controller tests", func() {
 			test.VerifyCert(secret, render.ManagerSecretKeyName, render.ManagerSecretCertName, dnsNames...)
 		})
 
-		It("should reconcile if existing user-supplied cert has the expected DNS names", func() {
-			// Create a manager cert secret with DNS names that include the
-			// expected DNS names.
-			dnsNames := append(expectedDNSNames, "manager.example.com", "192.168.10.22")
-			secret, err := render.CreateOperatorTLSSecret(
-				nil, render.ManagerTLSSecretName, render.ManagerSecretKeyName, render.ManagerSecretCertName, render.DefaultCertificateDuration, nil, dnsNames...)
+		It("should reconcile if operator-managed cert exists and user supplied a custom cert", func() {
```

Comment:
I don't understand this statement. How can both the operator-managed cert and a user supplied one exist?